app/assets/stylesheets/utilities/_overrides.scss

```diff
@@ -111,3 +111,13 @@
   margin-top: 0;
   background-color: govuk-colour("light-grey");
 }
+
+.govuk-tag-s .govuk-tag {
+  font-size: 70%;
+  vertical-align: middle;
+}
+
+h1 .govuk-tag-l {
+  font-size: 60%;
+  vertical-align: middle;
+}
```

Comment:
instead of this, we can just add font classes to the elements.

https://design-system.service.gov.uk/styles/typography/#font-override-classes

Reply:
Absolutely. I completely forgot that was a thing. Thanks!

Reply:
I think make all tags small - the big tags look a bit weird next to the small